src/test/java/org/jenkinsci/plugins/workflow/DynamicEnvironmentExpanderTest.java

```diff
@@ -145,4 +152,64 @@
         }
     }
 
+    @Issue("JENKINS-51170")
+    @Test public void perStepEnvironment() {
+        story.addStep(new Statement() {
+            @Override public void evaluate() throws Throwable {
+                WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "p");
+                // p.setDefinition(new CpsFlowDefinition("echo(/before VAR=$VAR/); " + StepEnvAdder.class.getCanonicalName() + ".value = 'after'; echo(/after VAR=$VAR/)", false));
```

Comment:
Any reason why the commented-out code is still here? Left over from some debugging session?